xyz-hub-service/src/main/resources/openapi.yaml

```diff
@@ -6,7 +6,7 @@ info:
   contact: {}
   version: 1.0.0
 servers:
-  - url: /hub
+  - url: http://localhost:8080/hub
```

Comment:
When being deployed on a different server, will this work as well?

Reply:
yes